addons/sales_team/static/src/css/sales_team.css

```diff
@@ -2,15 +2,13 @@
 .openerp .oe_kanban_view .oe_kanban_crm_salesteams {
   /* Customize width and height of kanban according bootstrap3 */
   width: 357px;
-  min-height: 254px !important;
-  /* End of customize */
-  cursor: default;
+  min-height: 80px !important;
 }
-.openerp .oe_kanban_view .oe_kanban_salesteams_avatars img {
-  width: 30px;
+.openerp .oe_kanban_view .oe_kanban_crm_salesteams .oe_kanban_content div:first-child {
+  margin-right: 0px;
 }
-.openerp .oe_salesteams_kanban_content {
-  margin-right: 16px;
+.openerp .oe_kanban_view .oe_kanban_crm_salesteams .oe_sparkline_bar {
+  margin-top: -6px;
```

Comment:
"sparkline bars should be aligned" has been done in another task (says BST). So rebase your branhc and remove this ;)